subprojects/docs/src/docs/release/notes.md

```diff
@@ -148,6 +148,17 @@ This avoids a 2nd request for the JAR file when the POM is missing, making depen
 
 You can also use this to programmatically opt-in to using the new Gradle metadata format for one repository using `metadataSources { gradleMetadata() }`.
 
+### Specifying reasons for dependency declaration and resolution rules
+
+In complex builds, dependency resolution results can be hard to interpret. Sometimes the reason why a [dependency declaration](userguide/customizing_dependency_resolution_behavior.html) or a [rule](userguide/inspecting_dependencies.html#sec:dependency_declaration_reasons) was added to the build script can get lost. To improve on this situation, we extended all the corresponding APIs with the capability to define a _reason_ for each declaration or rule. These reasons are shown in dependency insight reports and error messages if the corrsponding declaration or rule influenced the resolution result. In the future, they will also be shown in build scans.
```

Comment:
typo in _corresponding_ (missing the 'e')

Reply:
fixed